src/p4p/server/raw.py

```diff
@@ -170,12 +207,43 @@ def post(self, value, **kws):
         Any keyword arguments are forwarded to the NT wrap() method (if applicable).
         Common arguments include: timestamp= , severity= , and message= .
         """
+        # Intercept all arguments that start with 'handler_post_' and remove them from
+        # the arguments that go to the wrap and send them instead to the handler.post()
+        post_kws = {x: kws.pop(x) for x in [y for y in kws if y.startswith("handler_post_")]}
+
         try:
             V = self._wrap(value, **kws)
         except: # py3 will chain automatically, py2 won't
             raise ValueError("Unable to wrap %r with %r and %r"%(value, self._wrap, kws))
+
+        # Guard goes here because we can have handlers that don't inherit from 
+        # the Handler base class
+        try:  
+            self._handler.post(self, V, **post_kws)
+        except AttributeError:
+            pass
+
         _SharedPV.post(self, V)
```

Comment:
Implementing the `persist.py` example has made me think there may be a case to split the `post()` callback into two. The one implemented in this PR that changes values before they reach this `_SharedPV.post(self, V)` and one which is called afterwards. The `after_post()` callback would then only be supplied the updated state of the PV and would not be triggered by unsuccessful posts.